tools/AzureFunctionsJavaWorker.nuspec

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+  <metadata>
+    <id>Microsoft.Azure.Functions.JavaWorker</id>
+    <title>Microsoft Azure Functions Java Worker</title>
+    <tags>java azure-functions azure</tags>
+    <version>1.1.0-beta1-$version$</version>
```

Comment:
What does this version mean? Is it required to match the Java worker JAR version?

Reply:
This is the version of the NuGet. It's not required to match the .jar. We can version them independently, if we wish.